.github/workflows/argilla.yml

```
@@ -24,7 +24,7 @@ jobs:
   build:
     services:
       argilla-server:
-        image: argilladev/argilla-hf-spaces:develop
+        image: argilladev/argilla-hf-spaces:pr-5685
```

TODO: Change before merge
```suggestion
        image: argilladev/argilla-hf-spaces:develop
```